<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindowSettingsClass</class>
 <widget class="QWidget" name="ChatWindowSettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatWindowSettings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="openBox">
     <property name="text">
      <string>Open tab if received message</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="eventsBox">
     <property name="text">
      <string>Don't show events if message window is open</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="typingBox">
     <property name="text">
      <string>Send typing notifications</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="contentBox">
     <property name="text">
      <string>Don't show message content in notifications</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="removeBox">
     <property name="text">
      <string>Remove messages from chat view after:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="removeSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string> msg</string>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="namesBox">
     <property name="text">
      <string>Show names</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Timestamp:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="timestampComboBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>( hour:min:sec day/month/year )</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>( hour:min:sec )</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>( hour:min, full date ) </string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
